pkg/graph/resolvers/it_gov_task_statuses_requester.go

```diff
@@ -0,0 +1,40 @@
+package resolvers
+
+import "github.com/cmsgov/easi-app/pkg/models"
+
+//NOTE these functions are deterministic. Ideally when implementing we should separate the logic which obtains any database information from the methods that calculate the status
```

Comment:
It'll be a bit before I can review this fully, but I definitely want to 👍 this, it's a very good approach.